### PR TITLE
Added `getActiveMasternodeCount` RPC

### PR DESCRIPTION
**Pull request by siradji**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Added `getActiveMasternodeCount` RPC
#### Which issue(s) does this PR fixes?:

Fixes part of #48

### CODE REVIEW

File: packages/jellyfish-api-core/__tests__/category/masternode/getActiveMasternodeCount.test.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getActiveMasternodeCount', async () => {
+    jest.setTimeout(400000)
+    await client.masternode.createMasternode(await client.wallet.getNewAddress())
+
+    await container.generate(2016)
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/__tests__/category/masternode/getActiveMasternodeCount.test.ts:22]: I think you need to generate to the created masternode as well.

I think this is impossible to test without a multi-node setup, let's wait on this.

blocked by https://github.com/DeFiCh/jellyfish/issues/40

Comment (siradji) [reply packages/jellyfish-api-core/__tests__/category/masternode/getActiveMasternodeCount.test.ts]: @fuxingloh ,  I can now enable a masternode via `container.generate(nBlocks, address)` but i am yet to make it active. 

The active masternode count  does not change even after enabling a masternode. 

I thought enable means active but i guess i am wrong 

my question here is: What does active mean in regards to the `getActiveMasternode` RPC? 


```javascript
   await client.masternode.getActiveMasternodeCount() // 1

    const ownerAddress = await container.getNewAddress()

    await client.masternode.createMasternode(ownerAddress)
    await container.generate(20, ownerAddress) // Enables masternode

    await client.masternode.getActiveMasternodeCount() // 1
```

Comment (fuxingloh) [reply packages/jellyfish-api-core/__tests__/category/masternode/getActiveMasternodeCount.test.ts]: Need to wait for https://github.com/DeFiCh/jellyfish/pull/470 to be merged

Sorry I am a little too occupied now to review that

Comment (siradji) [reply packages/jellyfish-api-core/__tests__/category/masternode/getActiveMasternodeCount.test.ts]: Okay.

**Comment by fuxingloh** [comment]
@siradji update this PR and get it ready for merge.

**Comment by siradji** [comment]
@fuxingloh , i will closing this pull request. i have opened another one  to utilize the multi node setup

**Pull request closed without merging**